crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/controller/StatelessMeansAssessmentController.java

```diff
@@ -0,0 +1,101 @@
+            )
+    ) @Valid @RequestBody StatelessApiRequest meansAssessment) {
+        var apiAssessment = meansAssessment.getAssessment();
+        LocalDateTime date = apiAssessment.getAssessmentDate();
+        boolean hasPartner = apiAssessment.getHasPartner();
+        List<Income> income = meansAssessment.getIncome();
+        Map<AgeRange, Integer> childGroupings = getChildGroupings(apiAssessment.getDependantChildren());
+        if (apiAssessment.getAssessmentType() == StatelessRequestType.INITIAL) {
+            var result = statelessAssessmentService.initialOnly(date, hasPartner,
+                    apiAssessment.getCaseType(), apiAssessment.getMagistrateCourtOutcome(),
+                    childGroupings, income);
+            InitialMeansAssessment initialMeansAssessment = new InitialMeansAssessment()
+                    .withLowerThreshold(result.getInitialResult().getLowerThreshold())
+                    .withUpperThreshold(result.getInitialResult().getUpperThreshold());
+            var summary = new Summary().withOutcome(result.getOutcome());
+            var response = new StatelessApiResponse()
+                    .withInitialMeansAssessment(initialMeansAssessment)
+                    .withSummary(summary);
+
+            return ResponseEntity.ok(response);
+        } else {
+            var outgoings = meansAssessment.getOutgoings();
+            var result = statelessAssessmentService.initialAndFull(
+                    date, hasPartner,
+                    apiAssessment.getCaseType(), apiAssessment.getMagistrateCourtOutcome(),childGroupings,
+                    income, outgoings);
+            var initialResult = result.getInitialResult();
+            var fullResult = result.getFullResult();
+            InitialMeansAssessment initialMeansAssessment = new InitialMeansAssessment()
+                    .withLowerThreshold(initialResult.getLowerThreshold())
+                    .withUpperThreshold(initialResult.getUpperThreshold());
+            var summary = new Summary().withOutcome(result.getOutcome());
+            var response = new StatelessApiResponse()
+                    .withInitialMeansAssessment(initialMeansAssessment)
+                    .withSummary(summary);
+
+            if (fullResult != null) {
+                FullMeansAssessment fullMeansAssessment = new FullMeansAssessment()
+                        .withAdjustedLivingAllowance(fullResult.getAdjustedLivingAllowance())
+                        .withAdjustedIncomeValue(fullResult.getAdjustedIncomeValue())
+                        .withTotalAggregatedIncome(fullResult.getTotalAggregatedIncome())
+                        .withTotalAnnualAggregatedExpenditure(fullResult.getTotalAnnualAggregatedExpenditure())
+                        .withDisposableIncome(fullResult.getDisposableIncome());
+                response = response.withFullMeansAssessment((fullMeansAssessment));
+            }
```

Comment:
It's quite tricky to put it into the service - it would mean the service would know about the wire format (which I'm trying to avoid). I think I can reduce the code size though